### PR TITLE
Revisit global opt out, fix #231

### LINKED ISSUE #231: Don't claim general actions override specific ones
@michaelkleber points out that when https://w3ctag.github.io/privacy-principles/#opt-in-out says

> since a [global opt-out](https://w3ctag.github.io/privacy-principles/#dfn-global-opt-out) signal is reaffirmed automatically with every interaction, it will take precedence in terms of specificity over any general obtention of [consent](https://w3ctag.github.io/privacy-principles/#dfn-opt-in) by a site, and only superseded by specific [consent](https://w3ctag.github.io/privacy-principles/#dfn-opt-in) obtained through a deliberate action taken by the user with the intent of overriding their global opt-out.

it's confusing the fact that the technology sends a signal repeatedly, with the fact that the user only flipped a switch one time to cause it to do so. If a user makes a general setting in their browser, and a contrary setting on a specific website, it's the specificity of the *user's* actions that matter, not the number of times the browser repeats those actions, which means the website-specific setting overrides the global browser setting.

https://privacycg.github.io/gpc-spec/#legal-effects says that sites should disclose how they treat "conflicts between the signal and other specific privacy choices that the person has already made directly with the publisher", which avoids this inversion.

### PR DESCRIPTION
**Pull request by darobin**
This is intended to supersede #238.

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/296.html" title="Last updated on Jun 28, 2023, 6:02 PM UTC (c724a30)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/296/86d1c1f...c724a30.html" title="Last updated on Jun 28, 2023, 6:02 PM UTC (c724a30)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -677,11 +677,13 @@
...
-[=global opt-out=] signal is reaffirmed automatically with every interaction, it will take precedence
-in terms of specificity over any general obtention of [=consent=] by a site,
-and only superseded by specific [=consent=] obtained through a deliberate action taken by
-the user with the intent of overriding their global opt-out.
+their data not be sold or shared.) The [=user=] is effectively delegating the expression
+of their [=opt-out=] to their [=user agent=], which helps rectify [=automation asymmetry=].
+
+Under this model, a [=global opt-out=] signal should not be understood as a decision that a
+[=person=] made a while ago when they flipped a setting or chose to user a specific
+[=user agent=] but rather as a decision that they have chosen to automatically reaffirm with
```
Comment (darobin) [inline index.html:685]:
```suggestion
[=user agent=] but rather as a preference that they have chosen to automatically reaffirm with
```

File: index.html [hunk 2]
```diff
@@ -677,11 +677,13 @@
...
-their data not be sold or shared.) It should be noted that, since a
-[=global opt-out=] signal is reaffirmed automatically with every interaction, it will take precedence
-in terms of specificity over any general obtention of [=consent=] by a site,
-and only superseded by specific [=consent=] obtained through a deliberate action taken by
-the user with the intent of overriding their global opt-out.
+their data not be sold or shared.) The [=user=] is effectively delegating the expression
+of their [=opt-out=] to their [=user agent=], which helps rectify [=automation asymmetry=].
+
+Under this model, a [=global opt-out=] signal should not be understood as a decision that a
+[=person=] made a while ago when they flipped a setting or chose to user a specific
```
Comment (michaelkleber) [inline index.html:684]:
```suggestion
[=person=] made a while ago when they flipped a setting or chose to use a specific
```

**Comment by michaelkleber** [comment]
Thanks, I'm glad that this is now focused on the user's intent rather than irrelevant implementation details.  And I support being silent on the question of what supersedes what.

As always, I'm nervous about expansive claims about what the user intended, in a context where every user agent and/or other global privacy control provider can present a different UI preference experience (or even none at all).   But delegating this to the user agent is better than any other approach I can think of.

**Comment by darobin** [comment]
Right — that's why I tried to frame it in terms of the conceptual model of what a GOO is. The user is and will remain, ultimately, unknowable but I think that capturing the conceptual model of a GOO can help UAs make the right decisions in terms of what to present where.

**Review by jyasskin — APPROVED** [review]

**Pull request merged**